app/views/registrations/no_decision/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+  <% content_for :before_content, govuk_back_link(
+    text: "Back",
+    href: registrations_question_two_path)
+  %>
+    <h1 class="govuk-heading-l">You can register later</h1>
+    <p class="govuk-body">See our guidance for <a href="#">schools choosing an induction programme</a>.</p>
```

Comment:
From an accessibility point of view, it would be best if the entire thing were a link - but failing that, what's there already is better than just "our guidance".

Some screen reader users browse websites using links only (without the surrounding text), so if there's a link saying something like "see more", they have no idea what it's about.

Reply:
Makes sense, I guess that's a good reason to avoid links intertwined with text :P Oh well, something to think about later. Maybe something to flag to our designers.